homeassistant/components/weather/metoffice.py

```diff
@@ -0,0 +1,117 @@
+"""
+Support for UK Met Office weather service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.metoffice/
```

Comment:
Link needs to match the component otherwise the error message of the configuration check points to the wrong location. `sensor.*` -> `weather.*`